### PR TITLE
[wolftpm] update to 3.2.0

### PR DESCRIPTION
**Pull request by lealem47**

### CODE REVIEW

**Comment by lealem47** [comment]
@microsoft-github-policy-service agree company="wolfSSL"

File: ports/wolftpm/portfile.cmake [hunk 1]
```diff
@@ -1,11 +1,10 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO wolfssl/wolftpm
-    REF v2.7.0
-    SHA512 1b3e1bbd9f6e35226ff30cc6de7631fb420343a2ae4c8849d1477314a5fe8a95b99800fa17e10a30a9894041142748d654c913a5c4f4dc9eaad3f5157e405276
+    REF v3.2.0
+    SHA512 8cd63e0b62a2a72838c4ffabe00802d1297989ffbe3fbf20fc523160fc463ab6653ecab4aafb784a3f32d46a5efc71479e29dc1ca2fe1362573deb5b43525773
     HEAD_REF master
     PATCHES
-      cmake-patch.diff
```
Comment (JonLiu1993) [inline ports/wolftpm/portfile.cmake:8]: This patch has been applied to the latest version.

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

**Comment by lealem47** [comment]
ready for review

File: versions/w-/wolftpm.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "de626ef443c53f3e760a4c098c6da48c4a49c492",
+      "version": "3.2.0",
+      "port-version": 2
+    },
```
Comment (JonLiu1993) [inline versions/w-/wolftpm.json:7]:
```suggestion
```

Comment (JonLiu1993) [reply versions/w-/wolftpm.json:7]: Please remove this codes, then run command `./vcpkg x-add-version wolftpm --overwrite-version` and commit agagin.

Comment (lealem47) [reply versions/w-/wolftpm.json]: Done

**Comment by lealem47** [comment]
ready for review

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
Please convert the PR from Draft to Ready for review status.

**Pull request merged**